test/mocha/core/curlify.js

```diff
@@ -3,247 +3,335 @@ import Im from "immutable"
 import curl from "core/curlify"
 import win from "core/window"
 
-describe("curlify", function() {
```

Comment:
@tim-lai you should never do refactors inside bug fixes. At this point from this PR I cannot identify tests that are related to this fix. I'd suggest you revert the changes back and only fix/create relevant tests for this bug fix. Other refactoring should be part of different PR.

Reply:
reverted refactor. left in fixes for var/let/const/spaces